### PR TITLE
Upgraded NetApp Files SDK version to latest 1.13.0

### PR DESCRIPTION
**Pull request by AdelGhabboun**
1. Upgraded NetApp Files SDK version to 1.13.0
2. Upgraded target framework to .NET 5.0

### CODE REVIEW

**Review by hildurhodd — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,20 @@
 ## Azure NetApp Files Pool Change SDK Sample .NETCore
 
+[1.0.1 (2019-10-09)](#1.0.1 (2021-09-02))
```
Comment (paulomarquesc) [inline CHANGELOG.md:3]: >2019-10-09 [](http://example.com/codeflow?start=8&length=10)

Date here needs to be updated

**Review by paulomarquesc — CHANGES_REQUESTED** [review]
:clock1:

**Review by paulomarquesc — APPROVED** [review]
:shipit:

**Pull request merged**